amqp/src/AmqpClient.cpp

```diff
@@ -248,21 +223,22 @@ DeliveryState AmqpClient::unreceive()
 {
     std::lock_guard<std::mutex> lock(m_lock);
     auto                        deliveryState = DeliveryState::Unavailable;
-    if (m_receiver && m_receiver.active()) {
-        deliveryState = DeliveryState::Accepted;
-        m_receiver.close();
-        logDebug("Receiver Closed");
+    if (m_receiver) {
+        auto address = m_receiver.source().address();
```

Comment:
Sorry don't understand why we have only one address for unreceive... May be need to have a map of receiver